classes/Tools.php

```diff
@@ -2378,7 +2381,17 @@ public static function getMediaServer($filename)
             }
         }
 
-        if ($filename && self::$_cache_nb_media_servers && ($id_media_server = (abs(crc32($filename)) % self::$_cache_nb_media_servers + 1))) {
+        return self::$_cache_nb_media_servers > 0;
+    }
+
+    /**
+     * @param string $filename
+     *
+     * @return string
+     */
+    public static function getMediaServer($filename)
+    {
+        if ($filename && self::hasMediaServer() && ($id_media_server = (abs(crc32($filename)) % self::$_cache_nb_media_servers + 1))) {
```

Comment:
Do you have the reason why we have the crc32 here?

Reply:
Absolutely not ! I check log ;)

Reply:
After blame... It's historical !

Reply:
Dat sucks

Reply:
I have a small idea :
 - `abs(crc32($filename)` transform a string in a integer
 - and the modulo permits to not use always the same media server
 - but always use the same media server for a specific filename.

Not ?